chart/compass/templates/director-api-test.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: "testing.kyma-project.io/v1alpha1"
 kind: TestDefinition
 metadata:
-  name: {{ .Chart.Name }}-director
+  name: {{ .Chart.Name }}-director-api-tests
```

Comment:
Remove suffix `-tests`, as TestDefinition kind with this name is self-explanatory.

I would keep `-director` as it is